lazycoder1412/3SumSmaller.cpp

```diff
@@ -0,0 +1,18 @@
+class Solution {
+public:
+    int threeSumSmaller(vector<int>& nums, int target) {
+        sort(nums.begin(),nums.end());
+        int cnt=0;
+        for(int i=0;i<nums.size();i++){
+            int high=nums.size()-1;
```

Comment:
If nums.size() is reused in multiple locations, it is advisable to store it in `const int` variable named `nums_size`. Moreover using` nums.size() - 1` is dangerous because `size()` returns an unsigned integer. Thus if the vector `nums` is empty, `nums.size()-1` will underflow to a very large value.